packages/holocron/__tests__/externalRegistry.spec.jsx

```diff
@@ -128,6 +138,12 @@ describe('externalRegistry', () => {
         ]
       `);
     });
+
+    it('returns empty array when no externals', () => {
+      const moduleName = 'child-module-a';
+      externalRegistry.setModulesRequiredExternals({ moduleName, undefined });
```

Comment:
```suggestion
      externalRegistry.setModulesRequiredExternals({ moduleName, externals: undefined });
```
Not sure if this is what was intended?